cmd/sb/demo.html

```diff
@@ -0,0 +1,2069 @@
+<!doctype html>
```

Comment:
If you intend to keep this updated, probably move this file along with `runners/` and `configs/` into a `examples/` directory.
Otherwise, I'd suggest just use attachments in the github's PR for one time demo purposes.

Reply:
I've moved the `demo.html` alone with `runner/` and `config/` to `examples/`